docs/root/extending/extending.rst

```diff
@@ -19,7 +19,6 @@ types including:
 * :ref:`Stat sinks <arch_overview_statistics>`
 * :ref:`Tracers <arch_overview_tracing>`
 * :ref:`Request ID <arch_overview_tracing>`
-* :ref:`Transport sockets <envoy_v3_api_field_extensions.transport_sockets.tls.v3.CommonTlsContext.CertificateProvider.typed_config>`
```

Comment:
Why this is removed?

Reply:
Because that field is now marked as `not-implemented-hide`, which breaks the doc link.  And I don't think we should be advertising an unimplemented extension point in the docs anyway.

Reply:
`Transport sockets` are definitely implemented, this should be linked to https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/transport_socket/transport_socket.html instead.

Reply:
I can't figure out how to get RST to give me a link to that page, but I've re-added the bullet without the link.